virttest/qemu_vm.py

```diff
@@ -1777,7 +1777,7 @@ def sort_key(dev):
                 serial_name = prefix + str(len(self.virtio_ports))\
                     if prefix else serial
                 serial_params['serial_name'] = serial_name
-            if serial_params['serial_type'] == 'pci-serial':
+            if serial_params['serial_type'].startswith('pci'):
```

Comment:
@ldoktor I am not sure about the topology of `pci-serial-4x`, it should be pluged to `pcie-root-port` or `pci-bridge` ?

Currently, with a `False` in  `self._get_pci_bus(serial_params, 'serial', False)`, it will be plugged to pci-bridge, is this the expect result ?

Reply:
@yanan-fu  it looks this device should be plug to pci-bridge  device.

Reply:
@xutian ok, then the current change is ok for me, use `False` in `self._get_pci_bus(serial_params, 'serial', False)`.

Acked-by: Yanan Fu <yfu@redhat.com>